### ISSUE TITLE
No "Whatsapp" website is loaded in a "Facebook Container" when visited

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/contain-facebook#135]
**[Notes]:**
- This issue is reproducible with the following websites: "whatsapp.com", "https://www.whatsapp.com/", "https://www.whatsapp.com/" and "cdn.whatsapp.net".

**[Affected versions]:**
- Firefox 59.0.2 and above

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have a new Firefox profile with the "Facebook Container" add-on (1.4.0) installed.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
3. Navigate to the https://web.whatsapp.com/ website and observe the behavior.

**[Expected result]:**
- The page is opened in a "Facebook Container" tab.

**[Actual result]:**
- The page is opened in a normal tab.

**[Additional Notes]:**
- This issue is also reproducible if the website address is www or non-www, http or https, is accessed from the address bar or from the bookmarks.
- Attached a screen recording of the issue: 
![rec of the issue](https://user-images.githubusercontent.com/31243259/38727123-cbebf86c-3f14-11e8-9147-8dde925fb2a6.gif)

**Comment by TanviHacks** [comment]
@jonathanKingston or @pdehaan can you test this locally and provide your results?

**Comment by bakulf** [comment]
This seems already fixed by commit  221a2a80595b0bdec302f95f8c2ad3d9fa9ba843:
https://github.com/mozilla/contain-facebook/commit/221a2a80595b0bdec302f95f8c2ad3d9fa9ba843
@jonathanKingston can we mark this bug as fixed and close it?

**Comment by TanviHacks** [comment]
@Softvision-MariusComan The new code should have fixed this.  Can you checkout a new version and see if this is fixed?  Thanks!

**Comment by Softvision-MariusComan** [comment]
I have verified this issue and is no longer reproducible on the latest "Firefox Unbranded" build (60.0.1) and the latest Nightly build (62.0a1). The "Whatsapp" website is correctly loaded in a "Facebook Container" tab each time when it is visited. 
Considering the above I am closing this issue.

Verified on Windows 10 x64, Ubuntu 14.04 x64 and Mac 10.12 with the latest "Firefox Unbranded" build (60.0.1) and the latest Nightly (62.0a1 Build ID - 20180522220730) and using the latest version of the "Firefox Container" add-on (1.4.0 custom built on 23.05.2018).s
